src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentBindLoweringPass.cs

```diff
@@ -595,126 +591,76 @@ private IntermediateNode[] RewriteUsage(IntermediateNode parent, BindEntry bindE
             {
                 Content = ")",
                 Kind = TokenKind.CSharp,
             });
 
-            // Now rewrite the content of the change-handler node. There are two cases we care about
-            // here. If it's a component attribute, then don't use the 'BindMethods' wrapper. We expect
-            // component attributes to always 'match' on type.
+            // Now rewrite the content of the change-handler node. Since it's a component attribute,
+            // we don't use the 'BindMethods' wrapper. We expect component attributes to always 'match' on type.
             //
             // __value => <code> = __value
-            //
-            // For general DOM attributes, we need to be able to create a delegate that accepts UIEventArgs
-            // so we use BindMethods.SetValueHandler
-            //
-            // BindMethods.SetValueHandler(__value => <code> = __value, <code>) OR
-            // BindMethods.SetValueHandler(__value => <code> = __value, <code>, <format>)
-            //
-            // Note that the linemappings here are applied to the value attribute, not the change attribute.
-
-            string changeExpressionContent;
-            if (changeAttribute == null && format == null)
-            {
-                // DOM
-                changeExpressionContent = $"{ComponentsApi.BindMethods.SetValueHandler}(__value => {original.Content} = __value, {original.Content})";
-            }
-            else if (changeAttribute == null && format != null)
-            {
-                // DOM + format
-                changeExpressionContent = $"{ComponentsApi.BindMethods.SetValueHandler}(__value => {original.Content} = __value, {original.Content}, {format.Content})";
-            }
-            else
-            {
-                // Component
-                changeExpressionContent = $"__value => {original.Content} = __value";
-            }
             changeExpressionTokens.Add(new IntermediateToken()
             {
-                Content = changeExpressionContent,
-                Kind = TokenKind.CSharp
-            });              
+                Content = $"__value => {original.Content} = __value",
+                Kind = TokenKind.CSharp,
+            });
         }
 
         private void RewriteNodesForEventCallbackBind(
             IntermediateToken original,
             IntermediateToken format,
+            IntermediateToken culture,
             BoundAttributeDescriptor valueAttribute,
             BoundAttributeDescriptor changeAttribute,
             List<IntermediateToken> valueExpressionTokens,
             List<IntermediateToken> changeExpressionTokens)
         {
             // Now rewrite the content of the value node to look like:
             //
-            // BindMethods.GetValue(<code>) OR
-            // BindMethods.GetValue(<code>, <format>)
+            // BindMethods.GetValue(<code>, format: <format>, culture: <culture>)
```

Comment:
Are the parameter names necessary? I don't remember we do this in other places.

Reply:
yes, because the parameters will have default values. We want to allow specifying one or the other.